tools/windows-node-installer/README.md

```diff
@@ -24,86 +24,43 @@ make build-tools
 ```
 
 ## How to use it
-### Creating a Windows instance:
-
-```bash
-wni --help
-Creates and destroys Windows instance with an existing OpenShift cluster.
-
-Usage:
-  wni [command]
 
-Available Commands:
-  aws         Takes aws specific resource names from user
-  help        Help about any command
+The `wni` requires resources including the kubeconfig of the OpenShift cluster, a provider specific credentials file 
+and account to create and destroy a Windows instance on the selected provider. To create an instance, `wni` also 
+requires image id, instance type, and ssh key. Some optional flags include directory path to 
```

Comment:
Given that you are being generic, remove `ssh key` as that is AWS specific. Azure does not nee one.